ui/ui-config/pages/index.tsx

```diff
@@ -40,13 +40,16 @@ export default function Page(props) {
           target: null,
         });
       } else if (navigationMode === "config:authenticated" && !currentStep) {
-        const {
-          models: [model],
-        } = await client.request<Actions.ModelsList>("get", "/models", {
-          limit: 1,
-          order: [["name", "asc"]],
-        });
-        if (model) {
+        const { models } = await client.request<Actions.ModelsList>(
+          "get",
+          "/models",
+          {
+            limit: 1,
+            order: [["name", "asc"]],
+          }
+        );
+        if (models?.length > 0) {
+          const [model] = models;
```

Comment:
Was this throwing if `models.length` was  0?

Reply:
It will probably throw based on the current implementation because if there is a fetch error the `errorHandler` swallows it and we get a `{}` as a response.